### PR TITLE
Simplify jenkinsfile

### PR DESCRIPTION
**Pull request by iocanel**

### CODE REVIEW

**Comment by chirino** [comment]
Why not merge in the syndesis-pipeline-library into the mono repo?  Seems it would be easier to keep it in sync /w the Jenkins fil.

File: build.sh [hunk 1]
```diff
@@ -111,34 +103,34 @@ function init_options() {
...
   pushd verifier
-  "${MAVEN_CMD}" $MAVEN_CLEAN_GOAL install $MAVEN_PARAMS
+  mvn $MAVEN_CLEAN_GOAL install $MAVEN_OPTS
   popd
 }
 
 function runtime() {
   pushd runtime
-  "${MAVEN_CMD}" $MAVEN_CLEAN_GOAL install $MAVEN_PARAMS
+  mvn $MAVEN_CLEAN_GOAL install $MAVEN_OPTS
```
Comment (chirino) [inline build.sh]: please continue using MAVEN_CMD

File: build.sh [hunk 2]
```diff
@@ -101,7 +93,7 @@ function init_options() {
 
   if [ -n "$NAMESPACE" ]; then
       echo "Namespace: $NAMESPACE"
-      MAVEN_PARAMS="$MAVEN_PARAMS -Dfabric8.namespace=$NAMESPACE"
+      MAVEN_OPTS="$MAVEN_OPTS -Dfabric8.namespace=$NAMESPACE"
```
Comment (chirino) [inline build.sh]: Please don't rename.

**Comment by iocanel** [comment]
The changes to the `build.sh` where not intended and have been completely removed.

**Comment by chirino** [comment]
Jenkins kinda busted:

```
[INFO] --- git-commit-id-plugin:2.2.2:revision (default) @ core ---
Cannot contact syndesis-pr-355-12-q9dt5-5jmcb: java.io.IOException: remote file operation failed: /home/jenkins/workspace/syndesis_PR-355-JHKCMKEE6R677EXAYFIJF77B3JH3KBNHRIJVCJYD7YUJSOLU5B6A at hudson.remoting.Channel@1215e2e:JNLP4-connect connection from ip-10-1-12-1.ec2.internal/10.1.12.1:45098: hudson.remoting.ChannelClosedException: Remote call on JNLP4-connect connection from ip-10-1-12-1.ec2.internal/10.1.12.1:45098 failed. The channel is closing down or has closed down
```

**Pull request merged**